games.py

```diff
@@ -181,18 +182,21 @@ def query_player(game, state):
     game.display(state)
     print("available moves: {}".format(game.actions(state)))
```

Comment:
Alphabeta_search **cannot** have print statements in it, and cannot query the user. Rather, there can be a player who is a user; that player does the `input('Your move? ')` and does the `print`s.

Reply:
The print statements are currently inside query_player.